### ISSUE TITLE
Stage.contentsScaleFactor always 1 on Linux

### ISSUE THREAD

**Issue by pol2095** [airsdk/Adobe-Runtime-Support#1123]
## Description

Stage.contentsScaleFactor always 1 on Linux (Ubuntu 20 and Fedora 34)

I added in the app descriptor :
`<requestedDisplayResolution>high</requestedDisplayResolution>`

my app is very small vs the same app on Windows or Mac.

Thanks.

**Comment by pol2095** [comment]
Any news about this problem ?

**Comment by ajwfrost** [comment]
Hi @pol2095  - we had been looking into this and were trying to get the application scaling to work but ran into some issues.. I'll see whether the developer looking at this has had any further luck..

**Comment by pol2095** [comment]
Air app use gtk2, Ubuntu app using scaleFactor seems use gtk3.

**Comment by ajwfrost** [comment]
Yes that's what the developer has told me, he was looking at whether there are any cunning workarounds we can try to apply to get it to manually scale. But we can also look at whether it's feasible to update to gtk3 I guess..

**Comment by pol2095** [comment]
Any news about this Feature ?

Using Fractional Scaling it's possible to solve the problem (tried on Ubuntu).

**Comment by ajwfrost** [comment]
We are progressing on the GTK3 work, so once that's done we can set this up properly..

**Comment by pol2095** [comment]
why did you choose gtk3 and not gtk4 ?

**Comment by pol2095** [comment]
Any news of GTK3 on AIR SDK for Linux ?
GTK2 seems not working correctly on Linux Kernel 6.8 or later.
Why did you choose GTK3 and not GTK4 ?

**Comment by ajwfrost** [comment]
Hi

Sorry for the delay - just spoke with the guy who had done this, it's pretty much there but needs to be integrated into our latest codebase and then a lot of testing...

The switch from GTK2 to GTK3 looked a lot easier than the switch from GTK3 to GTK4! So mostly it was to make it easier to implement the changes; we didn't actually need any of the functionality from GTK4.

Our plan at the moment is to put this change in to our 51.2 branch and then probably push out a beta of it, as there are quite a few changes going in there. So we'll also push out another (lower-risk) update on 51.1 in the meantime..

thanks

**Comment by pol2095** [comment]
Any news about  GTK3 ?

**Comment by ajwfrost** [comment]
The work is done for this, we are planning to enable it in 51.2 - which has a lot of other changes too.. this has probably slipped now until early January, we may try and push it out as a beta initially to get some results from real-world scenarios.
